spec/models/source_spec.rb

```diff
@@ -5,7 +5,7 @@
   it { is_expected.to validate_presence_of(:url) }
 
   describe '.for_accepted_teams' do
-    let!(:accepted_team)   { create :team, kind: %w(sponsored).sample }
+    let!(:accepted_team)   { create :team, kind: %w(full_time).sample }
```

Comment:
Can you please remove this whole thing around creating an array with a single element and then taking a "sample" from it? I think it's fine to just to with the string `full_time` here.